hadoop-common-project/hadoop-common/pom.xml

```diff
@@ -342,6 +342,14 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
```

Comment:
What's the reason of adding the netty dependencies here?

Reply:
The secured ZK Client uses Netty, hence the dependency is included in the pom.xml. Added netty-handler and netty-transport-native-epoll dependency to the pom.xml based on ZOOKEEPER-3494 - "No need to depend on netty-all (SSL)". Adding this info to the Jira.

Reply:
Cool, thanks :)